### DISCUSSION TITLE
Updated to latest  TFT_eSPI Library - Still cant go larger than 320x170.  Should be 320x240.

### DISCUSSION THREAD

**Discussion by dookie454** [Bodmer/TFT_eSPI#2037] [Q&A - Sprites]
Hello, GREAT library!  Been using it for while, no problems with standard fonts, on two different pages,  decided to make a third page and include a Sprite in my ILI9341_DRIVER 240x320 display.  I got my Sprite Graph working somewhat, but ran into two problems..   

1) I am limited to: 320x170... if I make the 170 any larger and the entire sprite screen doesn't work, just stays blank with whatever I had on it from the previous screen.  I need 320x240, that should be the size of my screen.  With 170, I get fhe full width but the height is only 60%, the bottom of the sprite is cut off.    Code:   sprite.createSprite(320, 170); 

#2 shown below is fixed.. so ignore this!  
2) I decided to update to this latest library to see if that would fix the sprite problem, now I get Errors, these are them:
   Error 1:   'class TFT_eSPI' has no member named 'getTouch'.   Code:    if (tft.getTouch(&x, &y))
   Error 2:   'class TFT_eSPI' has no member named 'calibrateTouch'.  Code:  tft.calibrateTouch(calData, TFT_MAGENTA, TFT_BLACK, 15);
   Error 3:   'class TFT_eSPI' has no member named 'setTouch'.  Code:   tft.setTouch(calData);

Any help with these would be hugely appreciated!!!   

LATEST UPDATE:  Fixed Problem #2 (compile errors) by updating the files Lebo77 mentioned,  but still the big problem is still here, I cannot get the full screen working, limited to 320x170, nothing higher works.  Should be 320x240.

**Comment by Lebo77** [comment]
You need to re-set your user config file. You overwrote it when you
upgraded.

**Reply by dookie454** [reply]
well I tried the new examples included in the new library, the simple ones like Touch_Calibrate..  and they error for the same reasons.  So,  what went wrong here? 

Update..   I did copy the settings I had in User_Setup.h,  which was only the pin configuration, and yes, that seemed to fix it!  I didnt think that part would matter!!   THANKS SO MUCH!!!!!!!!!!!!!!!!!!

**Reply by dookie454** [reply]
Well, still having the 320x170 limitation,  I cant get anything larger than 170 to work .  Any ideas??

**Comment by Bodmer** [comment]
Sprite sizes are limited by the RAM available, which varies depending on the processor and available RAM. Mire info here:
https://github.com/Bodmer/TFT_eSPI#sprites
